File: abc028/src/b.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    string s;
    cin >> s;
    string AF = "ABCDEF";
    
    int c[6];
    for (int i = 0; i < 6; i++) {
      c[i] = count(s.begin(), s.end(), AF[i]);
    }
    string ans = to_string(c[0]);
    for (int i =1; i < 6; i++) {
      ans += " " + to_string(c[i]);
    }
    cout << ans << endl;
    return 0;
 }
     
    

File: abc028/src/bin/b.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();
    let mut s = String::new();
    stdin.lock().read_line(&mut s).unwrap();
    
    let af = "ABCDEF".chars().collect::<Vec<_>>();
    let mut c = [0; 6];
    for ch in s.chars() {
        for i in 0..6 {
            if ch == af[i] {
                c[i] += 1;
            }
        }
    }
    
    let ans = c.iter().map(|&x| x.to_string()).collect::<Vec<_>>().join(" ");
    println!("{}", ans);
}